packages/cli/.yo-rc.json

```diff
@@ -1415,7 +1415,7 @@
         "foreignKeyName": {
           "type": "String",
           "required": false,
-          "description": "Destination model foreign key name",
+          "description": "Destination model foreign key name (optional, do only specify if you understand the usage)",
```

Comment:
I'm good with specifying this as optional, however, frankly, I'm not sure if it's helpful to tell the user specify the value only "if you understand the usage". :)

Reply:
You are right. It doesn't provide any information to the user. The reason I want a change like this is that when the foreign key is supposed to be the default, it _shouldn't_ be provided. 

So, I think setting it to the following would be good: "Destination model foreign key name (optional, provide only when there is a custom foreign key".

Reply:
sounds good. If you could update the description accordingly, I can help merge this PR. thanks!